src/interfaces/app/user.ts

```diff
@@ -102,4 +103,5 @@ export interface UserCreationAttributes {
     pushNotificationToken?: string;
 
     trust?: AppUserTrustCreation;
+    overwrite?: boolean;
```

Comment:
this interface only contains model-related variables. `overwrite` isn't part of the User model or any model associated with it.
I see you've used this because of `authenticate`. As an alternative, I suggest changing `authenticate` to `authenticate(user: UserCreationAttributes & { overwrite?: boolean })` or `authenticate(user: UserCreationAttributes, overwrite: boolean =  false)` (I prefer the latter).